.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,31 @@
-<!-- Describe your pull request here -->
+**Describe your pull request here**
 
 ----
 
 Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).
 
-* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title
-* New recipes not directly relevant to the biological sciences need to be submitted to [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda
-* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please add the `please review & merge` label.
+* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
+* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
+* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
 * Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.
 
 <details>
-  <summary>BiocondaBot commands</summary>
-      
+  <summary>Please make use of the following BiocondaBot commands whenever appropriate:</summary>
+
 Everyone has access to the following BiocondaBot commands, which can be given in a comment:
 
-  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
-  * `@BiocondaBot please add label` will add the `please review & merge` label.
-  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
+  * `@BiocondaBot please update`
+     causes the BiocondaBot to merge the master branch into a PR.
```

Comment:
To make it more succinct, we can leave out the "will" part (unimportant semantics..).